Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -151,12 +178,20 @@ internal struct ReflectionServiceData: Sendable {
   }
 
   internal func nameOfFileContainingExtension(
-    named extendeeName: String,
+    extendeeName: String,
     fieldNumber number: Int32
   ) -> String? {
     let key = ExtensionDescriptor(extendeeTypeName: extendeeName, fieldNumber: number)
     return self.fileNameByExtensionDescriptor[key]
   }
+
+  internal func extensionsFieldNumbersOfType(named typeName: String) -> [Int32]? {
```

Comment:
Optional collections are rarely the right thing to do. Are we sure that we should return `[Int32]?` here vs `[Int32]`?

Reply:
I was following the pattern used for the other requests too, meaning that the functions of the registry could return nil, when the thing we are requesting for doesn't exist and the Reflection Service itself "decides" whether or not it should throw an error / send an error response. Should I change this?